cli/tests/test_config_schema.rs

```diff
@@ -0,0 +1,71 @@
+use std::process::Command;
+use std::process::Stdio;
+
+fn taplo_check_config(file: &str) {
+    if Command::new("taplo")
```

Comment:
(No action needed) I would prefer for this to also error out if the CI env var is set and taplo is not found, but it seems like we don't do this for `gpg`. So, I guess we should add it to both in a separate PR.

Reply:
Good idea, thank you!